process/transactionProcessor.go

```diff
@@ -1520,3 +1521,14 @@ func (tp *TransactionProcessor) getTxPoolNonceGapsFromObserver(
 
 	return &nonceGapsResponse.Data.NonceGaps, true
 }
+
+func applySortOnScrs(txWithSCRS *transaction.ApiTransactionResult) *transaction.ApiTransactionResult {
+	if txWithSCRS == nil {
+		return nil
+	}
+
+	sort.Slice(txWithSCRS.SmartContractResults, func(i, j int) bool {
+		return txWithSCRS.SmartContractResults[i].Hash < txWithSCRS.SmartContractResults[j].Hash
```

Comment:
we don't have here at this level the info related to `ExecutionOrder`? can't we order by execution order?